### PR TITLE
change image_thr to det_bbox_thr

### PR DESCRIPTION
**Pull request by jin-s13**
1) I replaced the variable name `image_thr` in cfg files with `det_bbox_thr` to avoid misunderstanding.
2) I removed the unused variable `bbox_thr` in cfg files.

### CODE REVIEW

**Comment by innerlee** [comment]
What was `bbox_thr` designed for

File: mmpose/datasets/datasets/top_down/topdown_aic_dataset.py [hunk 1]
```diff
@@ -53,13 +53,12 @@ def __init__(self,
 
         self.use_gt_bbox = data_cfg['use_gt_bbox']
         self.bbox_file = data_cfg['bbox_file']
-        self.image_thr = data_cfg['image_thr']
+        self.det_bbox_thr = data_cfg['det_bbox_thr']
```
Comment (innerlee) [inline mmpose/datasets/datasets/top_down/topdown_aic_dataset.py:56]: Help user to migrate?

```python
if `image_thr` in data_cfg:
    warning blablah
```

Comment (jin-s13) [reply mmpose/datasets/datasets/top_down/topdown_aic_dataset.py:56]: agree.

**Comment by jin-s13** [comment]
> What was `bbox_thr` designed for

Not used actually.

File: mmpose/datasets/datasets/top_down/topdown_aic_dataset.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def __init__(self,
 
         self.use_gt_bbox = data_cfg['use_gt_bbox']
         self.bbox_file = data_cfg['bbox_file']
-        self.det_bbox_thr = data_cfg['det_bbox_thr']
+        self.det_bbox_thr = data_cfg.get('det_bbox_thr', 0.0)
         self.use_nms = data_cfg.get('use_nms', True)
```
Comment (innerlee) [inline mmpose/datasets/datasets/top_down/topdown_aic_dataset.py:57]: missing warning for this

**Review by innerlee — APPROVED** [review]

**Pull request merged**